### PR TITLE
Add dev.edfloreshz.Blueprint

### PR DESCRIPTION
**Pull request by edfloreshz**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by barthalion** [comment]
bot, build dev.edfloreshz.Blueprint

**Comment by edfloreshz** [comment]
I forgot I had a git dependency, let me handle that and get back to you.

File: dev.edfloreshz.Blueprint.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+    "app-id" : "dev.edfloreshz.Blueprint",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
```
Comment (hfiguiere) [inline dev.edfloreshz.Blueprint.json:4]:
```suggestion
    "runtime-version" : "44",
```

**Comment by hfiguiere** [comment]
what's the relation with https://jwestman.pages.gitlab.gnome.org/blueprint-compiler/ ?

**Comment by edfloreshz** [comment]
They're separate projects.

**Comment by hfiguiere** [comment]
the name is confusing then

**Comment by edfloreshz** [comment]
They do fundamentally different things

**Comment by edfloreshz** [comment]
The only two matches you would get to Blueprint on Flathub are Workbench and this app, none of which are named the same.

**Comment by edfloreshz** [comment]
I just realized this app is probably not going into Flathub, it requires access to the host system to install packages via package managers.

Is there a way to make this work? As far as I know it would require using `flatpak-spawn` to run it outside the sandbox.

**Pull request closed without merging**